### PR TITLE
Fix: Remove pureLayout form MediaPreviewView

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Translate MediaPreviewView to Swift.
Change the argument of `pinToSuperView` form `constant` to `inset` to align to other constraint helper methods

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        }
+
+        titleLabel.pinToSuperview(anchor: .top, inset: 12)
+        titleLabel.pinToSuperview(anchor: .leading, inset: 12)
+
+        providerImageView.pinToSuperview(anchor: .top, inset: 15)
+        providerImageView.pinToSuperview(anchor: .trailing, inset: 12)
+
+        titleLabel?.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        providerImageView.leadingAnchor.constraint(greaterThanOrEqualTo: titleLabel.trailingAnchor, constant: 8)
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift]: Is this constraint activated?

File: Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    private func setupLayout() {
+        [contentView, containerView, previewImageView, overlayView, titleLabel, providerImageView, playButton].forEach() {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+
+        [contentView, containerView, previewImageView, overlayView].forEach() {
+            $0.fitInSuperview()
+        }
+
+        titleLabel.pinToSuperview(anchor: .top, inset: 12)
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift]: Would be nicer to activate all these constraints in on go.

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift]: The `pinToSuperview` helper method creates and activates the new constraints.

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift]: I know but it's more performant to activate all constraints in on go. As discussed here https://github.com/wireapp/ios-architecture/issues/47

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/MediaPreview/MediaPreviewView.swift]: I refactored it and activate all constraints in a batch.

**Pull request merged**